content/docs/practices/naming.md

```diff
@@ -26,7 +26,7 @@ A metric name...
  * <code><b>http</b>\_request\_duration\_seconds</code>
    (for all HTTP requests)
 * ...must have a single unit (i.e. do not mix seconds with milliseconds, or seconds with bytes).
-* ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers).
+* ...should use SI base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers).
```

Comment:
The kg is a SI base unit and the byte is not covered by SI units so this isn't really accurate.